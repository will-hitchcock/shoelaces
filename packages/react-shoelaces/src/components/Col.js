import styled from 'styled-components'
import { col } from 'shoelaces'

export const Col = styled.div`${col}`

export default Col
