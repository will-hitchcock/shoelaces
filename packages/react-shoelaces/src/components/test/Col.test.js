import Col from '../Col'

describe('<Col />', () => {
  it('should render a <Col />', () => {
    expect(shallow(<Col />)).toMatchSnapshot()
  })
})
