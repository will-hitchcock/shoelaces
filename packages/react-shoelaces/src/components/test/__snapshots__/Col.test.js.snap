// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Col /> should render a <Col /> 1`] = `
<div
  className="sc-bdVaJa bIezVD"
/>
`;
